### DISCUSSION TITLE
图片下载失败

### DISCUSSION THREAD

**Discussion by HMUNER** [ShunCai/QZoneExport#53] [Q&A]
试过chrome和edge
下载用过aria2，浏览器和助手内部
相册和说说的图片全部下载失败

**Comment by ghost** [comment]
我也下载失败了，怎么办

**Comment by ShunCai** [comment]
现在还是失败吗？

**Reply by ghost** [reply]
我换成360浏览器就不会下载失败，休息一下用EDGE再试试

**Reply by ShunCai** [reply]
> 我换成 360 浏览器就不会下载失败，休息一下用 EDGE 再试试

Chrome报错么？我优先兼容的就是Chrome。:cold_sweat:

**Reply by ShunCai** [reply]
> 我换成 360 浏览器就不会下载失败，休息一下用 EDGE 再试试

帮我看下Chrome控制台有没有错误信息？:smile:

**Reply by ghost** [reply]
> > 我换成 360 浏览器就不会下载失败，休息一下用 EDGE 再试试
> 
> 帮我看下Chrome控制台有没有错误信息？😄

我现在暂时不方便看，出错的是：所有说说图片下载失败

**Comment by LioDon** [comment]
我空间数据量特大，所以一直用迅雷。1.1.2版刚试过，可能是因为我空间数据量太大的原因，抓取到的数据不全，尤其是相册，16000+张照片只抓取不到1w……我忘了去年7月用1.0.8的时候是怎么把所有照片都备份下来的了……好像是一次只选一部分，分了两三次……

**Reply by ghost** [reply]
我从来不用这个软件备份QQ空间相册，我相册也不需要备份，所以帮不了你
